hooks/useChat.tsx

```diff
@@ -9,9 +9,14 @@ import useConversations from "./useConversations";
 import { useCsrfToken } from "@/packages/shared/src/hooks";
 import { useQueryClient } from "@tanstack/react-query";
 import useInitialMessages from "./useInitialMessages";
+import { Address } from "viem";
 
 const useChat = () => {
-  const { address, connectWallet } = useConnectWallet();
+  const { address, authenticated, ready } = useConnectWallet() as {
+    address: Address | undefined;
+    authenticated: boolean;
+    ready: boolean;
+  };
```

Comment:
Would updating the `export` of `useConnectWallet` to include `types` result in fewer code changes than needing to type cast all usages of the hook?